apps/hip/migrations/0003_auto_20210206_2119.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.1.5 on 2021-02-06 21:19
```

Comment:
Can we squash these migrations into 1? (To do this, delete all the migration files and re-run `manage.py makemigrations`)